indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -473,23 +469,23 @@
       });
     },
 
-    layout: function() {
-      this.gutter.runLayout();
-    },
-
     threadDeleted: function(view) {
       this.threadViews = _.without(this.threadViews, view);
       this.visibleThreads = _.without(this.visibleThreads, view);
-      var ix = this.gutter.items.indexOf(view);
-      if (ix > -1) {
-        this.gutter.items.splice(ix, 1);
-      }
-      this.layout();
+
+      document.querySelector('.active-thread-number-container .active-number').innerText = "";
+      document.querySelector('.active-thread-number-container .active-divider').innerText = "";
+      this.counts.set({
+        threads: this.visibleThreads.length,
```

Comment:
this is to fix I bug I discovered. the total thread count does not update when deleting a thread. Checked on `master` does the same thing